src/domain/auth/authenticate.ts

```diff
@@ -50,6 +48,19 @@ export const authenticateWithBackend = (
 
     dispatch(fetchTokenSuccess(res.data));
   } catch (error) {
-    dispatch(fetchTokenError(error));
+    toast(i18n.t('authentication.errorMessage'), {
+      type: toast.TYPE.ERROR,
+    });
+    try {
+      // This is a workaround that can save us until we can fix silentRenew().
```

Comment:
Would be better great if we create a ticket in the balcklog for this fix and mention it here, so it won't slip through the cracks.

Reply:
we already have it - VEN-378

Reply:
Cool, so can you mention it in the comment?

Reply:
done